projects/questionset-editor-library/src/lib/components/match/match.component.spec.data.ts

```diff
@@ -0,0 +1,114 @@
+export const mockOptionData = {
+  editorOptionData: {
+    question: "<p>Match the following with appropriate answer?</p>",
+    options: [
+      {
+        left: "<p>Lotus</p>",
+        right: "<p>Flower</p>",
+      },
+      {
+        left: "<p>Mango</p>",
+        right: "<p>Fruit</p>",
+      },
+    ],
+    templateId: "mtf-vertical",
+    correctMatchPair: [{ "0": 0 }, { "1": 1 }],
+    numberOfOptions: 4,
+  },
+  prepareMtfBody: {
+    templateId: "default",
```

Comment:
change the mock value of `templateId` to either 'mtf-horizontal' or 'mtf-vertical'

Reply:
completed